latex/delegation.tex

```diff
@@ -507,7 +512,9 @@ \subsection{Stake Pool Rules}
         \varUpdate{\var{poolParams}} & \varUpdate{\union}
                                     & \varUpdate{\{\var{hk} \mapsto \poolParam{c}\}} \\
        \var{retiring} \\
-       \var{avgs}
+       \var{avgs} \\
+       \varUpdate{\var{cs}} & \varUpdate{\union}
+                            & \varUpdate{\{\var{hk} \mapsto 0\}} \\
```

Comment:
in `OCERT` we use `VKey` and not its hash, should be done here, too

Reply:
good catch! I think for consistency's sake we can use HashKey in OCert, so that the PStake keys are all hashkeys.

Reply:
just pushed a change to use hashkey in ocert